test/libyul/objectCompiler/nested_subobject_data_access.yul

```diff
@@ -0,0 +1,92 @@
+object "A" {
+    code {
+        sstore(0, datasize("B.C"))
+        sstore(1, datasize("B.C.dataC1"))
+        sstore(2, datasize("B.C.dataC2"))
+    }
+
+    data "dataA1" "abcdef"
+    data "dataA2" "123"
+
+    object "B" {
+        code {
+            sstore(0, datasize("B"))
+            sstore(1, datasize("dataB1"))
+            sstore(2, datasize("dataB2"))
+        }
+
+        data "dataB1" "abcdef"
+        data "dataB2" "123"
```

Comment:
I'd remove this part. We have it in `subobject_qualified_vs_unqualified_data_access.yul` and here it does not seem to add much.

Instead I'd add `dataoffset()` in `A` and `C`. It should work the same way `datasize()` does but would not hurt to have it covered.